src/plugins/data/public/ui/query_editor/editors/default_editor/index.tsx

```diff
@@ -34,6 +34,105 @@ export const DefaultInput: React.FC<DefaultInputProps> = ({
   provideCompletionItems,
   queryStatus,
 }) => {
+  // Create a wrapper for editorDidMount to add our custom status bar
+  const handleEditorDidMount = (editor: monaco.editor.IStandaloneCodeEditor) => {
+    // Call the original editorDidMount function
+    editorDidMount(editor);
+
+    // Add a custom status bar with the "Tab to insert, ESC to close window" message
+    const addCustomStatusBar = () => {
+      // Find the suggestion widget
+      const editorElement = editor.getDomNode();
+      if (!editorElement) return;
+
+      // Use a MutationObserver to detect when the suggestion widget is shown
+      const observer = new MutationObserver(() => {
+        // Check if the suggestion widget is visible
+        const suggestWidget = editorElement.querySelector('.monaco-editor .suggest-widget.visible');
+        if (!suggestWidget) return;
+
+        // Get the suggestion list container (the widget itself)
+        const suggestWidgetElement = suggestWidget as HTMLElement;
+
+        // Check if our custom status bar already exists
+        let statusBar = document.querySelector('.custom-suggest-widget-status-bar');
```

Comment:
This seems like a code smell if we're directly querying the entire DOM to mutate internal components to Monaco. Since Monaco is an incredibly widely used library, I'd imagine they have a supported API to add a status bar. Otherwise if Monaco changes any internal DOM structure (which seems likely over time) this functionality will break.